_articles/de/starting-a-project.md

```diff
@@ -1,302 +1,328 @@
 ---
 lang: de
-title: Starting an Open Source Project
-description: Learn more about the world of open source and get ready to launch your own project.
+title: Ein Open-Source-Projekt anfangen
+description: Erfahren Sie mehr über die Open-Source-Welt und machen Sie sich bereit, Ihr eigenes Projekt zu starten.
 class: beginners
 toc:
-  the-what-and-why-of-open-source: "The what and why of open source"
-  should-i-launch-my-own-open-source-project: "Should I launch my own open source project?"
-  launching-your-own-open-source-project: "Launching your own open source project"
-  naming-and-branding-your-project: "Naming and branding your project"
-  your-pre-launch-checklist: "Your pre-launch checklist"
+  was-ist-open-source-und-warum: "Was ist Open Source, und warum?"
+  soll-ich-mein-eigenes-open-source-projekt-starten: "Soll ich mein eigenes Open-Source-Projekt starten?"
+  ein-eigenes-open-source-projekt-starten: "Ein eigenes Open-Source-Projekt starten"
+  ihr-projekt-benennen-und-zur-marke-machen: "Ihr Projekt benennen und zur Marke machen"
+  ihre-checkliste-zur-startvorbreitung: "Ihre Checkliste zur Startvorbreitung"
 order: 2
 image: /assets/images/cards/beginner.png
 related:
   - finding
   - building
 ---
 
-## The "what" and "why" of open source
+## Was ist Open Source, und warum?
 
-So you're thinking about getting started with open source? Congratulations! The world appreciates your contribution. Let's talk about what open source is and why people do it.
+Sie denken also über einen Sprung in die Open-SourceWelt nach? Herzlichen Glückwunsch, die Welt weiß Ihren Beitrag zu schätzen: Lassen Sie uns darüber sprechen, was Open Source ist und warum sich Leute dafür engagieren.
 
-### What does "open source" mean?
+### Was genau bedeutet "Open Source"?
 
-When a project is open source, that means **anybody can view, use, modify, and distribute your project for any purpose.** These permissions are enforced through [an open source license](https://opensource.org/licenses).
+Ein Open-Source-Projekt kann von **jedem für jeden Zweck verwendet, angesehen, modifiziert und weiterverbreitet werden.** Diese Rechte werden [mittels einer Open-Source-Lizenz durchgesetzt](https://opensource.org/licenses).
```

Comment:
Could you append a `(Englisch)` to the link?

Reply:
Let's move this discussion into [the main PR](https://github.com/github/opensource.guide/pull/577). I'll append a task to the first post there.